torch/nn/modules/loss.py

```diff
@@ -1141,7 +1141,7 @@ class probabilities only when a single class label per minibatch item is too res
         >>>
         >>> # Example of target with class probabilities
         >>> input = torch.randn(3, 5, requires_grad=True)
-        >>> target = torch.randn(3, 5).softmax(dim=1)
+        >>> target = torch.randn(3, 5).softmax(dim=1).argmax(dim=1)
```

Comment:
This is expected. The example above shows a target with indices and this one shows the soft target version.

Reply:
ah! thanks.. my bad.. didn't check the env. was using an old version.